src/resilience/exec_space/openMP/OpenMPResParallel.hpp

```diff
@@ -127,41 +193,106 @@ class ParallelFor< FunctorType
+      
+      // Combine the duplicate views, majority vote not triggered due to CMAKE macro
+      success = KokkosResilience::combine_resilient_duplicates();
+
+      if (!success)
+      {
+        KokkosResilience::ResilientDuplicatesSubscriber::dmr_failover_to_tmr = true;
+        KokkosResilience::ResilientDuplicatesSubscriber::in_resilient_parallel_loop = true;
+	auto m_functor_1 = m_functor;
+        KokkosResilience::ResilientDuplicatesSubscriber::in_resilient_parallel_loop = false;
+
+        Impl::ParallelFor< decltype(m_functor) , surrogate_policy, Kokkos::OpenMP > closure2(m_functor_1 , wrapper_policy );
+
+        start=std::chrono::steady_clock::now();
+        KokkosResilience::inject_error_duplicates();
+        stop=std::chrono::steady_clock::now();
+        KokkosResilience::ErrorTimerSettings::elapsed_seconds = KokkosResilience::ErrorTimerSettings::elapsed_seconds + (std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start));
+        KokkosResilience::ErrorTimerSettings::total_error_time = KokkosResilience::ErrorTimerSettings::total_error_time + KokkosResilience::ErrorTimerSettings::elapsed_seconds;
+
+        success = KokkosResilience::combine_resilient_duplicates();
+        KokkosResilience::clear_duplicates_map();
+        KokkosResilience::ResilientDuplicatesSubscriber::dmr_failover_to_tmr = false;
+      }
+      KokkosResilience::clear_duplicates_map();
+#endif
+#ifdef KR_ENABLE_WRAPPER
+
+      // TMR with kernel fusion
+      // Functor is fused, with iterations bound to duplicated functors in 3x range
+      Impl::ParallelFor< decltype(wrapper_functor) , surrogate_policy, Kokkos::OpenMP > closure( wrapper_functor , wrapper_policy );
+
+      closure.execute();
+
+      const auto start{std::chrono::steady_clock::now()};
+      KokkosResilience::inject_error_duplicates();
+      const auto stop{std::chrono::steady_clock::now()};
+      KokkosResilience::ErrorTimerSettings::elapsed_seconds = KokkosResilience::ErrorTimerSettings::elapsed_seconds + (std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start));
+      KokkosResilience::ErrorTimerSettings::total_error_time = KokkosResilience::ErrorTimerSettings::total_error_time + KokkosResilience::ErrorTimerSettings::elapsed_seconds;
+
+      // Combine the duplicate views and majority vote on correctness
+      success = KokkosResilience::combine_resilient_duplicates();
+      KokkosResilience::clear_duplicates_map();
+#endif
 
-    if(success==0 && repeats == 0){
-      // Abort if 5 repeKokkos::abort(ated tries at executing failed to find acceptable match
+    if(success==0){
+      // Abort if no agreement in duplicates
       auto &handler = KokkosResilience::get_unrecoverable_data_corruption_handler();
-      handler(max_repeats);
+      handler(static_cast<size_t>(success));
```

Comment:
Can just pass 0 in specifically here since we never repeat -- and success will always be 0.